ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -17,14 +23,17 @@ namespace ReactNative.Views.Image
     /// <summary>
     /// The view manager responsible for rendering native images.
     /// </summary>
-    public class ReactImageManager : SimpleViewManager<Border>
+    public class ReactImageManager : BaseViewManager<Border, ReactImageShadowNode>
     {
         private readonly Dictionary<int, SerialDisposable> _disposables =
             new Dictionary<int, SerialDisposable>();
 
         private readonly Dictionary<int, List<KeyValuePair<string, double>>> _imageSources =
             new Dictionary<int, List<KeyValuePair<string, double>>>();
 
+        private Color? _tintColor;
```

Comment:
@kevinvangelder can you pick up where @rozele left off? Can we expose the ShadowNodes within the ViewManager? or does the dictionary make more sense? I'm curious how this is done in Android.